examples/nft-d/nft.go

```diff
@@ -190,7 +191,7 @@ func Transfer(to interop.Hash160, token []byte, data interface{}) bool {
 		ok = true
 	}
 	if !ok {
-		panic("unknown token")
+		panic("unknown token" + std.Base64Encode(token))
```

Comment:
I think it is better to unify encodings here (base64) and in CLI (hex). Errors from RPC are reported unchanged and it would be easier to work with a single format.

Reply:
I would love to, and it should be hex everywhere because we already have hex-encoded IDs inside `getNEP11Transfers` RPC result. But we don't have std's API to convert bytes to hex, only base64 and base58 are supported.

Maybe create an issue in the C# repo to support hex encoding\decoding in native StdLib?

Reply:
Hex is not that hard to do in a smart-contract library, but maybe we don't need it at all here.

Reply:
Removed here and fixed in other places.